Platforms/QemuQ35Pkg/PlatformPei/Platform.c

```diff
@@ -830,10 +830,8 @@ InitializePlatform (
     MmSettings.HeapGuardPolicy.Fields.MmPageGuard                    = 1;
     MmSettings.HeapGuardPolicy.Fields.MmPoolGuard                    = 1;
     DxeSettings.ImageProtectionPolicy.Fields.ProtectImageFromUnknown = 1;
-    // THE /NXCOMPAT DLL flag cannot be set using non MinGW GCC
- #ifdef __GNUC__
+    // THE /NXCOMPAT DLL flag is not set on Linux today, so do not block loading

```

Comment:
Is this still true? I though GenFw would auto set Nxcompat as long as the image was ready:

https://github.com/microsoft/mu_basecore/commit/da219919538b679d5cf7387f4eba6c20384bf868

Reply:
I can make this comment more clear. I mean Linux itself, not EFI images built on Linux :). I.e. grub and shim